data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/security/Credentials.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.knative.eventing.kafka.broker.core.security;
+
+import org.apache.kafka.common.security.auth.SecurityProtocol;
+
+import javax.annotation.Nullable;
```

Comment:
No, we've never used them.

Reply:
Are you suggesting to remove them?

Reply:
TBH I'm not much a fan of Nullable annotations. They're practically useless for documentation, because you can just add in the javadocs that the return value can be null, and they require an exponential amount of other annotations to make the automatic checks working. Let's remove this and, if we really want these annotations, let's discuss them in another issue.

Reply:
Done!